README.md

```diff
@@ -0,0 +1,117 @@
+# 🐝 HackUPC: Getting Started Guidelines
+
+Weather if you are a first time hacker or a veteran, we encourage you to review this guideline. It will give you a great perspective for getting the most out of your hackathon in HackUPC!
```

Comment:
```suggestion
Whether if you are a first time hacker or a veteran, we encourage you to review this guideline. It will give you a great perspective for getting the most out of your hackathon in HackUPC!
```